UPEI-Android/group-project-mindjogg
# Dev Authentication Screen Set up

# Pull Request name

Login API documentation

## Source 

## Description 

Login API documentation

### Summary of PR  

We need to approve the PR as soon as possible because it contains the basic authentication flow of the app

### Purpose 

Implemented the authentication screens and set up basic authentication flow of the app

## Review and Testing Steps

1. Pull the branch
2. Go to mindjogg folder 
3. run "npm start"
4. Go to sign in screen to authenticate (username: admin, password: password)
5. Exit the app and check if the app needs authentication again (do not sign out now)
6. Sign out and  check if the app allow you to enter the app
7. Go to sign up app and enter the details and submit check the entered data in the console


### Notes
The app does not make any api call yet.
